spearhead/variables.css

```diff
@@ -25,7 +25,7 @@
 	--global--content-width: 900px;
 	/* Line Height */
 	--global--line-height-base: 1;
-	--global--line-height-body: 1.7;
+	--global--line-height-body: 1.6;
```

Comment:
Sorry — I must've mistyped in the original issue. This should be `1.8`, not `1.6`.